src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs

```diff
@@ -61,6 +61,7 @@ public void IncorrectVariableThrowsArgument()
         }
 
         [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "Environment Variables monidification isn't supported in Appx.")]
```

Comment:
It is on our list of requests to allow user level modification. If you're disabling this test for that then, you should create an issue to enable it as soon as it's fixed (which may mean, as soon as we move to testing on RS3 - not sure)

Otherwise they fix it (presumably) and we never test that ever.